docs/controls/AC.md

```diff
@@ -1,10 +1,8 @@
-# Reusable Component Library System Security Plan
-
-# NIST SP 800-53 Revision 4
```

Comment:
Best to preserve the standard names; there may be several within a certification and keeping them separated is good